manager/web-ui/src/reducers.js

```diff
@@ -195,8 +197,18 @@ function exec(state = initialExecState, action) {
 
 function pcap(state = initialPcapState, action) {
     switch (action.type) {
+    case PCAP_FETCHING:
+        return Object.assign({}, state, {
+          fetching: true,
+        });
+    case PCAP_FETCHED:
+        return Object.assign({}, state, {
+          fetching: false,
+	  fetched: true,
+          available: action.available,
+        });
     case PCAP_SELECTED:
-        return { ...initialPcapState,
```

Comment:
I'm not sure why this was here -- @JustinAzoff does this look OK or do I need to reset some other state here?

Reply:
I'm not really sure.  That's only even used in one or two places.  If uploading a pcap and switching between pcaps seems to work, it's probably fine.